dockerfiles/pyne_dag_openmc

```diff
@@ -0,0 +1,142 @@
+FROM debian:bookworm-slim
+
+ENV cores=18
```

Comment:
Should this be a build argument instead? Not guaranteed that all users will have 18 cores available